Quantum-Control-Applications/Superconducting/Two-Flux-Tunable-Transmons/Use-Case-2_Two-Qubit-Randomized-Benchmarking/configuration.py

```diff
@@ -0,0 +1,552 @@
+        "minus_y90_I_wf_q1": {"type": "arbitrary", "samples": minus_y90_I_wf_q1.tolist()},
+        "minus_y90_Q_wf_q1": {"type": "arbitrary", "samples": minus_y90_Q_wf_q1.tolist()},
+        "readout_wf_q1": {"type": "constant", "sample": readout_amp_q1},
+        "x90_I_wf_q2": {"type": "arbitrary", "samples": x90_I_wf_q2.tolist()},
+        "x90_Q_wf_q2": {"type": "arbitrary", "samples": x90_Q_wf_q2.tolist()},
+        "x180_I_wf_q2": {"type": "arbitrary", "samples": x180_I_wf_q2.tolist()},
+        "x180_Q_wf_q2": {"type": "arbitrary", "samples": x180_Q_wf_q2.tolist()},
+        "minus_x90_I_wf_q2": {"type": "arbitrary", "samples": minus_x90_I_wf_q2.tolist()},
+        "minus_x90_Q_wf_q2": {"type": "arbitrary", "samples": minus_x90_Q_wf_q2.tolist()},
+        "y90_I_wf_q2": {"type": "arbitrary", "samples": y90_I_wf_q2.tolist()},
+        "y90_Q_wf_q2": {"type": "arbitrary", "samples": y90_Q_wf_q2.tolist()},
+        "y180_I_wf_q2": {"type": "arbitrary", "samples": y180_I_wf_q2.tolist()},
+        "y180_Q_wf_q2": {"type": "arbitrary", "samples": y180_Q_wf_q2.tolist()},
+        "minus_y90_I_wf_q2": {"type": "arbitrary", "samples": minus_y90_I_wf_q2.tolist()},
+        "minus_y90_Q_wf_q2": {"type": "arbitrary", "samples": minus_y90_Q_wf_q2.tolist()},
+        "readout_wf_q2": {"type": "constant", "sample": readout_amp_q2},
+    },
+    "digital_waveforms": {
+        "ON": {"samples": [(1, 0)]},
+    },
+    "integration_weights": {
+        "cosine_weights": {
+            "cosine": [(1.0, readout_len)],
+            "sine": [(0.0, readout_len)],
+        },
+        "sine_weights": {
+            "cosine": [(0.0, readout_len)],
+            "sine": [(1.0, readout_len)],
+        },
+        "minus_sine_weights": {
+            "cosine": [(0.0, readout_len)],
+            "sine": [(-1.0, readout_len)],
+        },
+        "rotated_cosine_weights_q1": {
+            "cosine": [(np.cos(rotation_angle_q1), readout_len)],
+            "sine": [(-np.sin(rotation_angle_q1), readout_len)],
+        },
+        "rotated_sine_weights_q1": {
+            "cosine": [(np.sin(rotation_angle_q1), readout_len)],
+            "sine": [(np.cos(rotation_angle_q1), readout_len)],
+        },
+        "rotated_minus_sine_weights_q1": {
+            "cosine": [(-np.sin(rotation_angle_q1), readout_len)],
+            "sine": [(-np.cos(rotation_angle_q1), readout_len)],
+        },
+        "rotated_cosine_weights_q2": {
+            "cosine": [(np.cos(rotation_angle_q2), readout_len)],
+            "sine": [(-np.sin(rotation_angle_q2), readout_len)],
+        },
+        "rotated_sine_weights_q2": {
+            "cosine": [(np.sin(rotation_angle_q2), readout_len)],
+            "sine": [(np.cos(rotation_angle_q2), readout_len)],
+        },
+        "rotated_minus_sine_weights_q2": {
+            "cosine": [(-np.sin(rotation_angle_q2), readout_len)],
+            "sine": [(-np.cos(rotation_angle_q2), readout_len)],
+        },
+    },
```

Comment:
The sign of the rotated wieghts was actually wrong and I corrected it in the last update
        "rotated_cosine_weights_q1": {
            "cosine": [(np.cos(rotation_angle_q1), readout_len)],
            "sine": [(np.sin(rotation_angle_q1), readout_len)],
        },
        "rotated_sine_weights_q1": {
            "cosine": [(-np.sin(rotation_angle_q1), readout_len)],
            "sine": [(np.cos(rotation_angle_q1), readout_len)],
        },
        "rotated_minus_sine_weights_q1": {
            "cosine": [(np.sin(rotation_angle_q1), readout_len)],
            "sine": [(-np.cos(rotation_angle_q1), readout_len)],
        },
        "rotated_cosine_weights_q2": {
            "cosine": [(np.cos(rotation_angle_q2), readout_len)],
            "sine": [(np.sin(rotation_angle_q2), readout_len)],
        },
        "rotated_sine_weights_q2": {
            "cosine": [(-np.sin(rotation_angle_q2), readout_len)],
            "sine": [(np.cos(rotation_angle_q2), readout_len)],
        },
        "rotated_minus_sine_weights_q2": {
            "cosine": [(np.sin(rotation_angle_q2), readout_len)],
            "sine": [(-np.cos(rotation_angle_q2), readout_len)],
        },